### PR TITLE
API Updates

### PR DESCRIPTION
**Pull request by richardm-stripe**
Codegen for openapi 1ad8d09.
r? @seang-stripe 
cc @stripe/api-libraries

## Changelog
* Add support for new resource `TaxCode`
* Add support for `automatic_tax` `Invoice` and`Checkout.Session`.
* Add support for `tax_behavior` on `Price`
* Add support for `tax_code` on `Product`
* Add support for `tax` on `Customer`
* Add support for `tax_type` enum on `TaxRate`

### CODE REVIEW

File: lib/Product.php [hunk 1]
```diff
@@ -33,6 +33,7 @@
  * @property null|\Stripe\StripeObject $package_dimensions The dimensions of this product for shipping purposes.
  * @property null|bool $shippable Whether this product is shipped (i.e., physical goods).
  * @property null|string $statement_descriptor Extra information about a product which will appear on your customer's credit card statement. In the case that multiple products are billed at once, the first statement descriptor will be used.
+ * @property null|string|\Stripe\TaxCode $tax_code A <a href="https://stripe.com/docs/tax/tax-codes">tax code</a> ID.
```
Comment (seang-stripe) [inline lib/Product.php:36]: This link is dead, is it going to be rolled out as part of this release?

**Review by seang-stripe — CHANGES_REQUESTED** [review]
Just one question that I would like to understand. Thanks!

File: lib/Product.php (hunk 1 above)
Comment (richardm-stripe) [reply lib/Product.php:36]: Yes. If you log in and your account is opted-in you'll be able to see it now.
ptal @seang-stripe

**Review by seang-stripe — APPROVED** [review]
LGTM!

**Pull request merged**